example/tests/test_energy_balance.py

```diff
@@ -3,6 +3,9 @@
 import pytest
 from math import pi
 
+from hypothesis import given
+from hypothesis.strategies import floats
+
```

Comment:
I was thinking that it may be good idea to add for people the required imports for property-based exercises to: 
- Suggest they should complete the exercise in this file  
- Suggest what they need for the exercise